src/features/classification/security-controls/utils.js

```diff
@@ -0,0 +1,178 @@
+// @flow
+import getProp from 'lodash/get';
+import isNil from 'lodash/isNil';
+
+import type { MessageDescriptor } from 'react-intl';
+import type { AccessPolicyRestrictions } from './flowTypes';
+
+import downloadRestrictionsMessageMap from './downloadRestrictionsMessageMap';
+import messages from './messages';
+import { ACCESS_POLICY_RESTRICTION, DOWNLOAD_CONTROL, LIST_ACCESS_LEVEL, SHARED_LINK_ACCESS_LEVEL } from './constants';
+
+const { SHARED_LINK, DOWNLOAD, EXTERNAL_COLLAB, APP } = ACCESS_POLICY_RESTRICTION;
+const { DESKTOP, MOBILE, WEB } = DOWNLOAD_CONTROL;
+const { BLOCK, WHITELIST, BLACKLIST } = LIST_ACCESS_LEVEL;
+const { COLLAB_ONLY, COLLAB_AND_COMPANY_ONLY } = SHARED_LINK_ACCESS_LEVEL;
+
+const getShortSecurityControlsMessage = (accessPolicyRestrictions: AccessPolicyRestrictions): ?MessageDescriptor => {
+    const { sharedLink, download, externalCollab, app } = accessPolicyRestrictions;
+    // Shared link and external collab restrictions are grouped
```

Comment:
`const sharing = sharedLink || externalCollab` for reuse

Reply:
Updated